src/js/views/patients/worklist/filters/date-filter_views.js

```diff
@@ -56,6 +59,80 @@ const DefaultDateView = View.extend({
   },
 });
 
+const ControllerView = View.extend({
+  template: hbs`
+    <button class="button-secondary--compact is-icon-only js-prev">{{far "angle-left"}}</button>
+    <div class="inl-bl" data-date-picker-region></div>
+    <button class="button-secondary--compact is-icon-only js-next">{{far "angle-right"}}</button>
+  `,
+  regions: {
+    datepicker: '[data-date-picker-region]',
+  },
+  ui: {
+    next: '.js-next',
+    prev: '.js-prev',
+  },
+  childViewTriggers: {
+    'click': 'click:date',
+  },
+  triggers: {
+    'click @ui.prev': 'click:prev',
+    'click @ui.next': 'click:next',
+  },
+  onRender() {
+    const DatePickerViewClass = this.getOption('datePickerViewClass');
+    this.showChildView('datepicker', new DatePickerViewClass({
+      model: this.model,
+    }));
+
+    this.getTooltips();
+  },
+  getTooltips() {
+    const tooltipMessages = this.getTooltipMessages();
+
+    new Tooltip({
+      message: tooltipMessages.prevMessage,
+      uiView: this,
+      ui: this.ui.prev,
+    });
+
+    new Tooltip({
+      message: tooltipMessages.nextMessage,
+      uiView: this,
+      ui: this.ui.next,
+    });
+  },
+  getTooltipMessages() {
```

Comment:
how about 2 functions here where this one is `if(this.model.get('selectedDate') return this._getTooltipMessage(this.model.dayjs('selectedDate'), 'day' 'MM/DD/YYYY');`

And the other is something like: 
```
return {
  prevMessage: dayjs(ts).subtract(1, unit).format(format),
  nextMessage: dayjs(ts).add(1, unit).format(format),
};
```